pliops/consumer.cc

```diff
@@ -221,12 +221,11 @@ RepStatus Consumer::Stop()
   return RepStatus();
 }
 
-RepStatus Consumer::GetState(ConsumerState& state, std::string& error)
+RepStatus Consumer::GetState(ConsumerState& state)
 {
   std::lock_guard<std::mutex> lock(state_mutex_);
   state = state_;
-  error = error_;
-  return RepStatus();
+  return rc_;
```

Comment:
I see what you did. But I think the return value meaning is different. It refers to the function call status, but the rc_ is only refers to the background errors.